public/index.html

```diff
@@ -40,6 +40,15 @@
           url(/assets/fonts/Karla-Regular.woff2) format('woff2'),
           url(/assets/fonts/Karla-Regular.woff) format('woff');
       }
+      @font-face {
+        font-display: swap;
+        font-family: 'Karla';
+        font-style: normal;
+        font-weight: 800;
+        src: local('Karla Bold'), local('Karla-Bold'),
+          url(/assets/fonts/Karla-Bold.woff2) format('woff2'),
+          url(/assets/fonts/Karla-Bold.woff) format('woff');
+      }
```

Comment:
This block of code needs to be added to `.storybook/preview-head.html` as well. See https://storybook.js.org/docs/react/configure/story-rendering#adding-to-head